mesonbuild/wrap/wraptool.py

```diff
@@ -89,21 +89,36 @@ def get_latest_version(name: str, allow_insecure: bool) -> T.Tuple[str, str]:
     version, revision = latest_version.rsplit('-', 1)
     return version, revision
 
-def install(options: 'argparse.Namespace') -> None:
-    name = options.name
-    if not os.path.isdir('subprojects'):
-        raise SystemExit('Subprojects dir not found. Run this script in your source root directory.')
+def install_one(name: str, allow_insecure: bool) -> None:
     if os.path.isdir(os.path.join('subprojects', name)):
-        raise SystemExit('Subproject directory for this project already exists.')
+        print(f'Subproject directory for {name} already exists.')
+        return
```

Comment:
This is an API change. Formerly, if a wrap already existed and the `meson wrap install` failed, we raised SystemExit, now we ignore the issue entirely.

I understand you don't want to raise SystemExit because other wraps should have a chance to be installed too, but in that case we should switch to raising a WrapException.